codewof/programming/content/en/celsius-to-fahrenheit/question.md

```diff
@@ -0,0 +1,8 @@
+# Celsius to fahrenheit
+
+Write a function `celsius_to_fahrenheit(temperature)` that takes a temperature in celsius, and **returns** the temperature converted into fahrenheit.
+The formula for converting celsius into fahrenheit is: Celsius x 9/5 + 32 = Fahrenheit.
```

Comment:
I made a couple of minor points on #69 about this, also consider using _degrees_ celsius and _degrees_ fahrenheit